src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -44,6 +44,34 @@ template <typename DataType>
 struct SqrtDetSpatialMetric : db::SimpleTag {
   using type = Scalar<DataType>;
 };
+/*!
+ * \brief Conformal factor used with a conformal spatial metric.
+ */
+template <typename DataType>
+struct SpatialConformalFactor : db::SimpleTag {
```

Comment:
Tags for conformal quantities are already available in `Elliptic/Systems/Xcts/Tags.hpp`. How about we move the `Tags::Conformal` prefix to `PointwiseFunctions/GeneralRelativity/Tags/Conformal.hpp`, and you use it to define tags with the appropriate power of the conformal factor in `Ccz4/Tags.hpp`?